opentracing-api/pom.xml

```diff
@@ -20,7 +20,7 @@
     <parent>
         <groupId>io.opentracing</groupId>
         <artifactId>parent</artifactId>
-        <version>0.31.1-SNAPSHOT</version>
+        <version>0.32.0-RC3-SNAPSHOT</version>
```

Comment:
Should we do an RC3?

Reply:
I think we are ready for an actual release - the latest (important) change from RC2 is making `Tracer` implement `Closable` ;)